board/safety/safety_ford.h

```diff
@@ -116,6 +116,22 @@ static bool ford_lkas_msg_check(int addr) {
       || (addr == MSG_IPMA_Data);
 }
 
+// Curvature rate limits
+const SteeringLimits FORD_STEERING_LIMITS = {
+  .angle_deg_to_can = 50000,  // 1 / (2e-5) rad to can
+  .angle_rate_up_lookup = {
+    {5., 25., 25.},
```

Comment:
should there only be two breakpoints like OP?

Reply:
`interpolate` does NOT detect the length properly lol, above 25 the output is 0, so blocked msgs